apps/events/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from apps.user.models import User 
+# Create your models here.
+
+
+class Events(models.Model):
+    
+    name = models.CharField(max_length=150, null=True)
+    description = models.CharField(max_length=255, null=True)
+    startTime = models.DateTimeField(auto_now_add=True)
+    endTime = models.DateTimeField(auto_now=True)
+    type = models.CharField(max_length=150, null=True)
```

Comment:
type is a key word in Python, consider changing to something else like category or group